charts/reports/templates/deployment.yaml

```
@@ -51,8 +51,10 @@ spec:
               mountPath: /usr/share/nginx/html
             - name: config
               mountPath: /etc/nginx/conf.d
+          {{- with .Values.resources }}
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          {{- end}}
```

```suggestion
          {{- with .Values.resources }}
          resources:
            {{- toYaml . | nindent 12 }}
          {{- end}}
```

Otherwise:

```
Error: Failed to render chart: exit status 1: Error: template: reports/templates/deployment.yaml:56:30: executing "reports/templates/deployment.yaml" at <.Values.resources>: nil pointer evaluating interface {}.resources
```

(worth a unit test with custom values to catch this error, then fix it with my suggestion)